1-org/README.md

```diff
@@ -40,7 +40,7 @@ You can choose not to enable the Data Access logs by setting variable `data_acce
     _TF_SA_EMAIL
     _STATE_BUCKET_NAME
     ```
-1. Copy terraform wrapper script `cp ../terraform-example-foundation/build/tf-wrapper.sh . `1. Copy terraform wrapper script `cp ../terraform-example-foundation/build/tf-wrapper.sh . ` to the root of your new repository  (modify accordingly based on your current directory). to the root of your new repository (modify accordingly based on your current directory).
+1. Copy terraform wrapper script `cp ../terraform-example-foundation/build/tf-wrapper.sh . ` to the root of your new repository  (modify accordingly based on your current directory). to the root of your new repository (modify accordingly based on your current directory).
```

Comment:
The phrase `to the root of your new repository (modify accordingly based on your current directory).` is repeated twice.

Reply:
Will open another PR for this issue. Thank you!